### ISSUE TITLE
Need ability to cache previous views

### ISSUE THREAD

**Issue by scottblinch** [alefort/getcrafty#2]
Clicking into a view and then using the corresponding action button to go back to the previous one reloads that view entirely anew instead of just bringing that old view's DOM back.

This is particularly problematic when searching locations by postal code. Going back to that view reloads the default geolocation behavior instead of the postal code results.

**Comment by alefort** [comment]
Can this be done with angular? Shouldn't we be able to do this with angular views?

**Comment by scottblinch** [comment]
Yep, it can be done with Angular. It just looks complicated :stuck_out_tongue:

**Comment by alefort** [comment]
We can tag team that I'm sure.

**Comment by alefort** [comment]
I know we ended up ripping this out, caching locations was problematic for folks who traveled lots.

Is this still something we want?

**Comment by alefort** [comment]
In perspective, loading a store page takes 500ms for domloaded, and 900ms for content loaded. Pretty snappy.
